### DISCUSSION TITLE
Upsert into SQL table by multiple conditions failure in prisma python? #702

### DISCUSSION THREAD

**Discussion by shsadegh** [prisma/prisma#17974] [Q&A]
How should I query over relation by multiple conditions?
for example I want to filter table rows with the below properties (all of them are unique and primary key):
column1: value1 (PK)
coulmn2: value2 (PK)
column3: value3 (PK)

**My written code:**
`
record = db.table.upsert(
       where={'column1': 1,
                    'column2': 2,
                    'column3': 3}
       data={
                    'create': {
                        ....
                    },
                    'update': {
                        ....
                    },
                  },
      )
`
**And the error that I encounterd :**
prisma.errors.FieldNotFoundError: Failed to validate the query: `Expected exactly one field to be present, got 3.

**Comment by nurul3101** [comment]
Hey @shsadegh 👋 

I would recommend you to open a GitHub Discussion in this repository: https://github.com/RobertCraigie/prisma-client-py
The maintainer of the prisma client python repository should be able to help you out.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
